### ISSUE TITLE
Tarea 4 P1 Actividad 6 Determinar las clases que más se confunden para cada red

### ISSUE THREAD

**Issue by sofiagiagnoni** [IIC2613-Inteligencia-Artificial-2022-1/Syllabus#119]
Hola!

Tengo una duda con la parte 6, ¿cómo puedo saber cuáles son las clases que más se confunden en cada una de las redes neuronales?
¿Se da por el accuracy, val_accuracy, test accuracy o algún otro valor?

Saludos

**Comment by SarahEverke** [comment]
Hola!
Con una matriz de confusión puedes visualizar cuáles son las clases que más se confunden, por ejemplo la matriz de confusión de [sklearn](https://scikit-learn.org/stable/modules/generated/sklearn.metrics.confusion_matrix.html) recibe como *input* los valores verdaderos y los que se predijeron.

**Comment by sofiagiagnoni** [comment]
Gracias!!
